src/collections/Events.ts

```diff
@@ -44,17 +44,44 @@ export const Events: CollectionConfig = {
           displayFormat: "dd.MM.yyyy",
         },
       },
+      validate: (value: string, options) => {
+        if (options.siblingData.start > value) {
+          return "Das Enddatum muss nach dem Startdatum sein.";
+        } else {
+          return true;
+        }
+      },
       required: true,
     },
     {
-      name: "timetable",
+      name: "shortDescription",
+      type: "text",
+      required: true,
+    },
+    {
+      name: "calendarCover",
       type: "relationship",
       relationTo: ["media"],
+      required: true,
     },
     {
-      name: "shortDescription",
+      name: "slug",
       type: "text",
-      required: true,
+      required: false,
+      unique: true,
+      admin: {
+        description: ({ value }: { value?: unknown }) =>
+          value
+            ? `Eventseite erreichbar unter: https://youngvision.org/events/${value ?? ""} Entferne den slug um keine Eventseit zu generieren.`
+            : "Es wird keine Eventseite erstellt, weil der slug leer ist."
+      },
+      validate(value: string, options) {
+        if (value && value.match(/^[a-z0-9][a-z0-9\-]+[a-z0-9]$/) == null) {
+          return "Es sind nur Kleinbuchstaben(a-z), Ziffern(0-9) und Bindestriche(-) erlaubt. Der slug muss mindestens 3 Zeichen lang sein. Bindestriche dürfen nich am Anfang oder Ende sein.";
```

Comment:
I think it would be nice to separate these error messages. So if the value is shorter than 3 characters, show that. If it doesn't match the regex, show the message about which characters are allowed.

If we do that it might be good to short circuit the validation with a check at the beginning if the slug is empty.